rustls/src/webpki/verify.rs

```diff
@@ -134,6 +135,15 @@ impl<'a> TryFrom<&'a CertificateDer<'a>> for ParsedCertificate<'a> {
     }
 }
 
+/// So that [`ParsedCertificate`] may transparently behave like a [`webpki::EndEntityCert`].
+impl<'a> Deref for ParsedCertificate<'a> {
+    type Target = webpki::EndEntityCert<'a>;
```

Comment:
Unfortunately, this still counts as a webpi type appearing in the public API. If we're only relying on this to get access to `subject_public_key_info()` I'd recommend just adding an inherent impl which forwards that method to the webpki method of the same name.

Reply:
That makes sense. I've opted to simply remove the impl in [b3577ab](https://github.com/rustls/rustls/pull/1954/commits/b3577abc6b66c77c0e5024adcd8248f8f70115f7) and access the inner cert via `.0` as done elsewhere in the repo